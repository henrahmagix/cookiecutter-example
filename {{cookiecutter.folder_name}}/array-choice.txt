Array option chosen was "{{cookiecutter.array_of_options}}"
