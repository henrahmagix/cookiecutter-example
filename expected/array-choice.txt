Array option chosen was "a"
